librato/alerts.py

```diff
@@ -129,16 +129,15 @@ def from_dict(cls, data):
         return obj
 
     def get_payload(self):
-        obj = {'condition_type': self.condition_type,
-               'metric_name': self.metric_name,
-               'source': self.source}
+        obj = {
+            'type': self.condition_type,
```

Comment:
The API exposes `type` not `condition_type` but both should actually be supported on POST. So, let's make the payload use `type`